src/cli/subcommands/car_cmd.rs

```diff
@@ -55,7 +55,7 @@ impl CarCommands {
     }
 }
 
-fn merge_car_streams<R>(
+pub fn merge_car_streams<R>(
```

Comment:
Not a big of importing methods defined in `cmd` into another crate. Perhaps move them into some shared location? That also goes for `dedup_block_stream`.

Reply:
Moved to `car_util.rs`